src/main/kotlin/no/nav/syfo/melding/MeldingService.kt

```diff
@@ -70,15 +70,13 @@ class MeldingService(
             PdfContent(it.pdf)
         }
 
-    private fun getBehandlerRefForConversation(conversationRef: UUID, personIdent: PersonIdent): UUID {
-        return getUtgaendeMeldingerInConversation(
-            conversationRef = conversationRef,
-            personIdent = personIdent,
-        )
-            .firstOrNull()
-            ?.behandlerRef
-            ?: throw IllegalStateException("Fant ikke behandlerRef for samtale $conversationRef, kunne ikke knyttes til melding fra behandler")
-    }
+    private fun getBehandlerRefForConversation(
+        meldingFraBehandler: MeldingFraBehandler,
+        personIdent: PersonIdent
+    ) = getUtgaendeMeldingerInConversation(
+        conversationRef = meldingFraBehandler.conversationRef,
+        personIdent = personIdent,
+    ).firstOrNull()?.behandlerRef
```

Comment:
Tenker vi fortsatt kan kaste exception her hvis `meldingFraBehandler` har annen type enn `HENVENDELSE_MELDING_TIL_NAV`

Reply:
Jeg tenkte at hvis det kommer en melding til NAV med en kjent conversationRef så skal den legge seg inn i den tråden (dvs at det antakelig er svar på en melding til behandler).

Reply:
Det gir mening! Men exception hvis man ikke finner `behandlerRef` for meldingstyper initiert av veileder tenker jeg er lurt da :smile: